packages/ontario-design-system-component-library/src/components/ontario-date-input/utils/date-validation-utils.ts

```diff
@@ -38,7 +38,7 @@ const isInvalidMonth = (value: string) => {
 /*
  * Year field should be a number, not negative, at length at least 4 digits
  */
```

Comment:
Consider extending the comment to explain that isInvalidYear validates numeric and range-bound conditions. The existing comment is more like supplemental text.

Reply:
Good catch, I'll update this!